autotst/calculator/orca.py

```diff
@@ -0,0 +1,206 @@
+        If found, it creates an attribute for that property.
+        """
+
+        # Assert AutoTST conformer is attached to Orca Calculator
+        assert self.conformer is not None,'Must provide an AutoTST conformer object'
+        assert isinstance(self.conformer,Conformer),'conformer must be an autotst conformer object'
+        
+        # Assign smiles or reaction label as label attribute
+        if isinstance(self.conformer, TS):
+            self.label = self.conformer.reaction_label
+        else:
+            self.label = self.conformer.smiles
+
+        # Replace problematic characters in temporary files and assign to base attribute
+        if '(' in self.label or '#' in self.label:
+            self.base = self.label.replace('(', '{').replace(')', '}').replace('#', '=-')
+        else:
+            self.base = self.label
+  
+        self.charge = self.conformer.rmg_molecule.getNetCharge()
+        self.mult = self.conformer.rmg_molecule.multiplicity
+
+        try:
+            self.coords = self.conformer.get_xyz_block()
+        except:
+            logging.warning('could not get coordinates of conformer...setting coords to None')
+            self.coords = None
+
+
+    def write_fod_input(self,directory=None):
+        """
+        Generates input files to run finite temperaure DFT to determine the Fractional Occupation number weighted Density (FOD number).
+        Uses the default functional, basis set, and SmearTemp (TPSS, def2-TZVP, 5000 K) in Orca.
+        See resource for more information:
+        Bauer, C. A., Hansen, A., & Grimme, S. (2017). The Fractional Occupation Number Weighted Density as a Versatile Analysis Tool for Molecules with a Complicated Electronic Structure. 
+        Chemistry - A European Journal, 23(25), 6150–6164. https://doi.org/10.1002/chem.201604682
+        """
+
+        # Make sure we have required properties of conformer to run the job
+        assert None not in [self.mult,self.charge,self.coords]
+        
+        # If directory is not specified, use the instance directory
+        if directory is None:
+            directory = self.directory
+        else:
+            if not os.path.exists(directory):
+                os.makedirs(directory)
+
+        # Path for FOD input file
+        outfile = os.path.join(directory,self.label+'_fod.inp')
+
+        # Write FOD input
+        with open(outfile, 'w+') as f:
+            f.write('# FOD anaylsis for {} \n'.format(self.label))
+            f.write('! FOD \n')
+            f.write('\n')
+            f.write('%pal nprocs {} end \n'.format(str(self.nprocs)))
+            f.write('%scf\n  MaxIter  600\nend\n')
+            f.write('%base "{}_fod" \n'.format(self.base))
+            f.write('*xyz {} {}\n'.format(self.charge, self.mult))
+            f.write(self.coords)
+            f.write('*\n')
+
+    def check_NormalTermination(self,path):
```

Comment:
Please change this to `check_normal_termination` to match the styling of methods in the rest of the codebase.